src/main/java/uk/gov/justice/hmpps/casenotes/config/CaseNoteContextInterceptor.kt

```diff
@@ -24,7 +24,7 @@ import uk.gov.justice.hmpps.casenotes.legacy.dto.UserDetails.Companion.NOMIS
 class CaseNoteContextConfiguration(private val caseNoteContextInterceptor: CaseNoteContextInterceptor) : WebMvcConfigurer {
   override fun addInterceptors(registry: InterceptorRegistry) {
     registry.addInterceptor(caseNoteContextInterceptor)
-      .addPathPatterns("/case-notes/**", "/case-notes/**/**")
+      .addPathPatterns("/case-notes/**", "/admin/case-notes/**")
```

Comment:
Took me a while to be confident but /case-notes/** matches _zero_ or more path segments and /case-notes/**/** matches _one_ or more. We don't need both and don't have any solely /case-notes endpoints so for our purposes, /case-notes/** is all we need